source/plugins/go/src/network_flow_logs.go

```diff
@@ -0,0 +1,372 @@
+package main
```

Comment:
We should check network flow logs code not executed for the windows if cluster has both Linux and windows nodes.

Reply:
it won't as IsNetworkFlowLogsEnabled will be false for windows